extensions/php/languages/php/tasks.json

```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "label": "php test $ZED_SYMBOL",
+    "command": "vendor/bin/phpunit",
+    "args": ["--filter=$ZED_SYMBOL"],
+    "tags": ["php-test"]
```

Comment:
*nitpick*: the [recently added](4eca7875ae25469c5a43f3f73f7160c861cba3c8) gleam test runner uses the label `gleam test ...` but, in that case, the command that is actually being run is `gleam test ...`. In this case, the label may be more accurately named `phpunit $ZED_SYMBOL`

Ditto for the tag name: perhaps `phpunit-test` would be a better tag name, in case someone wants to later add support for [alternate PHP test runners](https://pestphp.com/).

Reply:
Agree, will change that!